cardano-api/src/Cardano/Api/Tx.hs

```diff
@@ -108,6 +112,20 @@ import           Cardano.Api.SerialiseCBOR
 import           Cardano.Api.SerialiseTextEnvelope
 import           Cardano.Api.TxBody
 
+import qualified Cardano.Ledger.ShelleyMA.TxBody as Allegra
+
+-- | Get all the @Certificate@ from @Tx@.
+getTxCertificates :: forall era. Tx era -> Seq.StrictSeq Certificate
```

Comment:
We generally don't expose uses of `StrictSeq`. Is there a particular reason to use it here over `[]`?

Reply:
Just thought of trying to keep the original data structure in place without conversions in case somebody needs it.